src/test/regress/expected/multi_test_helpers.out

```diff
@@ -0,0 +1,86 @@
+-- File to create functions and helpers needed for subsequent tests
+-- create a helper function to create objects on each node
+CREATE FUNCTION run_command_on_master_and_workers(p_sql text)
+RETURNS void LANGUAGE plpgsql AS $$
+BEGIN
+     EXECUTE p_sql;
+     PERFORM run_command_on_workers(p_sql);
+END;$$;
```

Comment:
@anarazel : Yeah, I agree. It hasn't been a huge pain to ask them to do this in two steps, but it would definitely be an add-on.